lib/puppet/provider/cloudwatch_alarm/v2.rb

```diff
@@ -33,10 +33,15 @@ def self.prefetch(resources)
   read_only(:region, :alarm_actions)
 
   def self.alarm_to_hash(region, alarm)
-    response = autoscaling_client(region).describe_policies(
-      policy_names: alarm.alarm_actions
-    )
-    actions = response.scaling_policies.collect(&:policy_name)
+    actions = alarm.alarm_actions
+
+    unless actions.first.start_with?('arn:aws')
```

Comment:
Is it reasonable to only inspect the first action, or should all actions be inspected for `arn:aws` when collecting the `policy_name`?